addons/partner_autocomplete/tests/test_partner_autocomplete.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.tests import common
+
+
+class PartnerAutocompleteCase(common.SavepointCase):
+
+    def setUp(self):
+        super(PartnerAutocompleteCase, self).setUp()
+
+        Company = self.env['res.company']
+
+        self.company_1 = Company.create({
+            'name': "Test Company 1",
+        })
```

Comment:
That's... a lot of diff.
I mean, you could just do

`company_1 = self.env['res.company'].create({'name': "Test Company 1"})`
At the beginning of your "test_extract_company_domain" and be done.